ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -143,55 +157,109 @@ export class Send extends IronfishCommand {
       this.exit(1)
     }
 
-    if (fee == null) {
-      let suggestedFee = ''
-      try {
-        const response = await client.estimateFee({
-          fromAccountName: from,
-          receives: [
-            {
-              publicAddress: to,
-              amount: CurrencyUtils.encode(amount),
-              memo: memo,
-            },
-          ],
+    if (expiration !== undefined && expiration < 0) {
+      this.log('Expiration sequence must be non-negative')
+      this.exit(1)
+    }
+
+    let rawTransactionResponse: string
+    if (fee == null && feeRate == null) {
+      const feeRates = await client.estimateFeeRates()
+      const feeRateOptions = []
+
+      const createTransactionRequest: CreateTransactionRequest = {
+        sender: from,
+        receives: [
+          {
+            publicAddress: to,
+            amount: CurrencyUtils.encode(amount),
+            memo,
+            assetId,
+          },
+        ],
+        expiration: expiration,
+      }
+
+      const allPromises = [
+        client.createTransaction({
+          ...createTransactionRequest,
+          feeRate: feeRates.content.low,
+        }),
+        client.createTransaction({
+          ...createTransactionRequest,
+          feeRate: feeRates.content.medium,
+        }),
+        client.createTransaction({
+          ...createTransactionRequest,
+          feeRate: feeRates.content.high,
+        }),
+      ]
+
+      const createResponses = await Promise.all(allPromises)
+
+      if (feeRates.content.low) {
```

Comment:
I know `low` is optional in the response content, and I think it is beyond the scope of this PR to change that RPC endpoint, but isn't it unexpected if the response doesn't include `low`?

What will happen here if the response doesn't include low?

Reply:
I tested this case locally and I get `Invalid transaction fee rate, 0` from the `createTransaction` request above.

Reply:
Yeah the `Invalid transaction fee rate, 0` is expected since fee rate returned is 0. I changed the fee rate rpc to return value large than 0. Since 0 fee rate is not a valid value to use.